index.Rmd

```diff
@@ -13,13 +13,33 @@ output:
       toc: true
 ---
 
-```{r include=FALSE}
-# automatically create a bib database for R packages
-knitr::write_bib(c(
-  .packages(), "bookdown", "knitr", "rmarkdown"
-), "packages.bib")
+# Overview {-}
+
+This book introduces WDL Workflows on AnVIL.
+
+```{r, echo=FALSE}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1o2XnuMbqWVLf4XrsXolIQ7ulfnMlpJlrUxN0Y8aLIVQ/edit#slide=id.g1397c25e58c_0_2")
 ```
 
-# About this Book {-}
+## Learning Objectives {-}
+
+- Understand when WDL Workflows are the right tool
+- Run a Workflow on AnVIL
+- Write a WDL using Broad Methods Repository
+- Bring your own data to analyze
+- Customize your Docker environment
+- Join the conversation
+
+## Prerequisites {-}
```

Comment:
I wonder if it's worth trying to be consistent with what's in the [SARS book](https://jhudatascience.org/GDSCN_Book_SARS_Galaxy_on_AnVIL/index.html#skills-level). If so, I might structure it like :

```
::: {.notice}
_Resources_  
**Genetics**: No programming experience needed

_Programming skills_  
**Novice**: No programming experience needed
:::
```

Then learning objectives, and then later on down:

```
::: {.warning}
_Resources_  
**Funding**: Requires access to a Terra Billing Project
:::
```

Also. Wondering if a little programming experience would be helpful for WDL, eg.
`Beginner: minimal prior knowledge/skill helpful`

Reply:
Agree on trying to be consistent!  Happy to move Learning Objectives to the third position, but I'd like to avoid having two {.notice} if possible.  Two questions:
1. Do you think it's worth stating that it requires access to a Terra Billing Project on all the hands-on books?  Or is that just a given?
2. Thoughts on the friendlier title Skills Level vs more general Prerequisites (which would allow for things like Funding)?

Reply:
Re prerequisites: The original intention with this box is that instructors can compare among activities, not so much that it's a prerequisite per se.

Could the .notice go at the top of the next big section instead? Although I think as @KatherineCox mentioned in https://github.com/jhudsl/AnVIL_Template/pull/161 maybe this is kind of a given. I'm okay either way.